projects/map-cesium/package.json

```diff
@@ -21,8 +21,8 @@
   "dependencies": {
     "@dlr-eoc/services-map-state": "12.0.0-alpha.2",
     "@dlr-eoc/services-layers": "12.0.0-alpha.2",
-    "@cesium/engine": "^3.0.2",
-    "@cesium/widgets": "^3.0.2",
+    "@cesium/engine": "^6.1.0",
```

Comment:
Updating libraries in the major version could be considered a breaking change.

@voinSR and @MichaelLangbein I think we discussed this once. But with a lot of libraries in our repo, the versions could get very high very quickly. We have to decide how to deal with that. 

For now, it is ok because this library was only available as an alpha version.

Reply:
Yes, I wanted to update the versions before we create a new release of the libraries.